homeassistant/components/romy/manifest.json

```diff
@@ -0,0 +1,10 @@
+{
+  "domain": "romy",
+  "name": "ROMY Vacuum Cleaner",
+  "codeowners": ["@xeniter"],
+  "config_flow": true,
+  "documentation": "https://www.home-assistant.io/integrations/romy",
+  "iot_class": "local_polling",
+  "requirements": ["romy==0.0.5"],
```

Comment:
Not a blocker for this PR to get merged but some findings in your lib:

- https://github.com/xeniter/romy/blob/main/src/romy/__init__.py#L4 specify the version to be `0.0.2` even when `0.0.5` is installed. You can remove the version field if you don't need it.
- https://github.com/xeniter/romy/blob/main/src/romy/romy.py#L9-L11 are unused imports